packages/neutrino-preset-web/index.js

```diff
@@ -53,6 +56,11 @@ module.exports = (neutrino, opts = {}) => {
     }, options.babel)
   });
 
+  Object.assign(options.clean, opts.clean || {});
```

Comment:
I don't think you need these lines since you are already setting defaults on line 34. You should be able to safely delete L59-63.